### PR TITLE
Fixed android build with wireguard

### PR DESCRIPTION
**Pull request by soffokl**

### CODE REVIEW

File: cmd/connections_register_android.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ */
+
+package cmd
+
+import (
+	"github.com/mysteriumnetwork/node/core/node"
+)
+
+func (di *Dependencies) registerConnections(nodeOptions node.Options) {
+	di.registerOpenvpnConnection(nodeOptions)
```
Comment (tadovas) [inline cmd/connections_register_android.go]: This can be removed too. We redefine openvpn connection for android anyway

Comment (soffokl) [reply cmd/connections_register_android.go]: And only noop will be left here. :D
Should I remove both?

Comment (tadovas) [reply cmd/connections_register_android.go]: No. Noop should work on all oses (and it kind of works :D)

Comment (soffokl) [reply cmd/connections_register_android.go]: removed openvpn.

**Comment by tadovas** [comment]
@zolia @vkuznecovas - we need this to fix master build

**Pull request merged**